tests/slack_sdk/oauth/installation_store/test_amazon_s3.py

```diff
@@ -0,0 +1,270 @@
+import unittest
+
+import boto3
+from moto import mock_s3
```

Comment:
We have been using moto in bolt-python project already but this is the first test using it in this project https://github.com/spulec/moto#example-on-unittest-usage